code/modules/surgery/organs/augments_arms.dm

```diff
@@ -311,3 +311,20 @@
 
 /obj/item/organ/cyberimp/arm/hydraulic_blade/l
 	zone = BODY_ZONE_L_ARM
+
+/obj/item/organ/cyberimp/arm/botany
+	name = "botanical arm implant"
+	desc = "A rather simple arm implant containing tools used in gardening and botanical research."
+	items_to_create = list(/obj/item/cultivator, /obj/item/shovel/spade, /obj/item/hatchet, /obj/item/gun/energy/floragun, /obj/item/plant_analyzer, /obj/item/storage/bag/plants, /obj/item/storage/bag/plants/portaseeder)
+
+/obj/item/organ/cyberimp/arm/janitor
+	name = "janitorial tools implant"
+	desc = "A set of janitorial tools on the user's arm."
+	items_to_create = list(/obj/item/lightreplacer/cyborg, /obj/item/holosign_creator/janibarrier, /obj/item/soap/nanotrasen, /obj/item/reagent_containers/spray/cyborg/drying_agent, /obj/item/mop/advanced/cyborg, /obj/item/paint/paint_remover, /obj/item/reagent_containers/spray/cleaner)
+
+/obj/item/organ/cyberimp/arm/janitor/emag_act(mob/user)
+	for(var/datum/weakref/created_item in items_list)
```

Comment:
?

Reply:
I don't know what you mean. It works if that's what you're concerned about

Reply:
Its a pointless for loop that does absolutely nothing.

Reply:
Oh, right, it doesn't do anything 😅